README.md

```diff
@@ -1,31 +1,147 @@
+
+### Install Appium and related drivers & plugins
+
+The NPM package manager can install the Appium service package and all of its related 
+dependencies and utilities for you.
+
+Run this command from a terminal prompt in the project root directory where the 
+`package.json` file is stored.
+```bash
+npm install
+```
+
+### Install and configure Android Studio
+
+Installation and setup instructions for Android Studio here.
+
+## Usage
+
+You will need to configure the Applitools Eyes Appium SDK, and launch an Android device
+emulator on your local system with the example application installed before you run the
+example tests in this project.
+
+If you're testing your application using a mobile device grid, you should already know how
+to connect an Appium test to your test device.  The instructions in this document describe
+how to connect the tests to an emulator running on your computer.
+
+**Note:** whether you plan to test the example application using a local emulator or a
+remote device, you will still need to configure the Eyes Appium SDK as described below.
+
+### Set your APPLITOOLS_API_KEY
+
+First, 
+[retrieve your API key](https://applitools.com/docs/topics/overview/obtain-api-key.html) 
+for your user account from the Eyes Dashboard UI.  The Eyes Appium SDK will use this value 
+as an authentication token to upload visual UI checks to your Eyes test results dashboard.
+
+There are two ways to 
+[provide your Applitools API key to the Eyes SDK](https://applitools.com/tutorials/quickstart/web/selenium/java/basic#setting-applitools-variables).
+
+1. Store it the `APPLITOOLS_API_KEY` environment variable or `eyes.api_key` property.
+   The Eyes SDK will read the value from there automatically, and you do not need to set 
+   it explicitly in your test code.
+  
+2. You can hard-code your API key in the test source code by replacing 
+   [line 105 of the Hooks.java](src/test/java/AppiumTests/Hooks.java#L105) source file
+
+>> `eyes.setApiKey(System.getenv("APPLITOOLS_API_KEY"));`
+
+> with 
+
+>> `eyes.setApiKey("<YOUR_API_KEY_GOES_HERE>");`
 
-## Installing appium and related drivers & plugins
-`npm install`
+***Note:*** Your Applitools API key is a secret value.  The tests you run will
+use your API key as an authentication token to connect to the Eyes Service, so 
+**treat it like a password** and do not share it with anyone.
 
-## Running the tests
+**If you have hard-coded your API key in the example test code, do not to push the source code file that includes your secret API key to a public GitHub repo!**
+
+
+### Start an Android device emulator
+
+[Create an Android Virtual Device in Android Studio.](https://developer.android.com/studio/run/managing-avds)
+
+[Run an emulator in Andriod Studio.](https://developer.android.com/studio/run/managing-avds#emulator)
+
+
+### Install the example application
+
+This project's sampleApps directory contains an APK file for the example Android 
+Calculator native mobile application.
+
+To 
```

Comment:
@sbernheim - the test will automatically install the app in the device/emulator.